linera-views/src/common.rs

```diff
@@ -41,6 +41,51 @@ pub(crate) enum Update<T> {
     Set(T),
 }
 
+#[derive(Clone, Debug)]
+pub(crate) struct DeletionSet {
+    pub delete_storage_first: bool,
+    pub deleted_prefixes: BTreeSet<Vec<u8>>,
```

Comment:
`delete_storage_first == true` is basically equivalent to `deleted_prefixes == BTreeSet::from([vec![]])`, is it? Would it make sense to simplify this, or would it actually complicate things in the end?

Reply:
Yes, it is equivalent, but I would prefer to keep the boolean as it is used in the code and I think we would not gain in simplicity.